string name
string type
---
int32 result
